kornia/core/_backend.py

```diff
@@ -29,5 +29,6 @@
 # random
 rand = torch.rand
 
-# device
-Device = Union[str, device]
+# type alias
+Device = Union[str, device, None]
+Dtype = Union[torch.dtype, None]
```

Comment:
we can't have the Optional here (`None`) because torch 1.9.1 don't have complete stubs -- like, they support it on runtime, but do not have the right annotations. So, one option is remove the typing CI for `1.9.1`, what do you think @edgarriba ?

Reply:
Sounds good, I like pretty much the style with future annotations